src/vs/workbench/contrib/webview/browser/pre/main.js

```diff
@@ -277,6 +277,15 @@
 		 * @param {KeyboardEvent} e
 		 */
 		const handleInnerKeydown = (e) => {
+			// If the keypress would trigger a browser event, such as copy or paste,
```

Comment:
@deepak1556 I opted against using `before-input-event` since this problem also effects iframe based webviews